otel/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+
+services:
+  # OpenTelemetry Collector
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:latest
+    container_name: otel-collector
+    command: ["--config=/etc/otel-collector-config.yaml"]
+    volumes:
+      - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
+    ports:
+      - "4317:4317"   # OTLP gRPC receiver
+      - "4318:4318"   # OTLP HTTP receiver
+      - "8889:8889"   # Prometheus exporter
+      - "8888:8888"   # Metrics endpoint
+    depends_on:
+      - zipkin
+      - jaeger
+
+  # Jaeger
+  jaeger:
```

Comment:
You could try out SigNoz. It's amazing.

Reply:
Will try.
How the setup with Jaeger you see is for demonstration. It is up to the users to use the tracing tool that works best for them.